### ISSUE TITLE
elastic skin

### ISSUE THREAD

**Issue by grape6** [pimlie/authres_status#26]
in elastic skin authres_status icon is shown only after pressing 'envelope' button. Maybe better
to show it always.

**Comment by pimlie** [comment]
Unfortunately I am unable to get the elastic skin working, if you could do a PR that would be great. You could add a file `skins/elastic/authres_status.css` which should list all css required for it to work correctly in the elastic skin.

**Comment by grape6** [comment]
sorry, i'm not so familiar with elastic :)

**Comment by photoninger** [comment]
Playhing around with roundcube-1.4.0-rc2 and elastic:
* authres_status is working in messages_list
* as a noob with hooks in roundcube i don't think the hook ```message_headers_output``` can change the tag ```<div class="header-subject">``` which contains the from header in default mail view because it is not called until you click the envelope for detailed headers

**Comment by FlyveHest** [comment]
1.4.0 is now out, and elastic is the default skin going forward. If you could somehow find the time to update this great plugin, I think a lot of people would be grateful, me included :)

**Comment by SKBanan** [comment]
This seems to be only about localization, in "authres_status.php"

```
public function message_summary($p){
return array('content' => preg_replace('/(<span>\s*)()(From)/', '$1' . $this->img_status . ' $3', $p['content']));
```

replaced with: 
`return array('content' => preg_replace('/(<span>\s*)()('.$this->gettext('from').')/', '$1' . $this->img_status . ' $3', $p['content']));`

Now the status icon appears correctly in main "From" field, without the need to click message details

**Comment by photoninger** [comment]
> This seems to be only about localization, in "authres_status.php"
> 
> ```
> public function message_summary($p){
> return array('content' => preg_replace('/(<span>\s*)()(From)/', '$1' . $this->img_status . ' $3', $p['content']));
> ```
> 
> replaced with:
> `return array('content' => preg_replace('/(<span>\s*)()('.$this->gettext('from').')/', '$1' . $this->img_status . ' $3', $p['content']));`
> 
> Now the status icon appears correctly in main "From" field, without the need to click message details

Yes, it is about localization, I opend a pull request to change the regex: https://github.com/pimlie/authres_status/pull/48
